shopfloor_single_product_transfer_mobile/static/src/scenario/single_product_transfer.js

```diff
@@ -11,19 +11,30 @@ const SingleProductTransfer = {
     template: `
         <Screen :screen_info="screen_info">
             <template v-slot:header>
+                <template v-if="state_is('select_product')">
+                    <state-display-info :info="current_location_msg()" v-if="current_location()"/>
+                    <state-display-info :info="current_package_msg()" v-if="current_package()"/>
+                </template>
```

Comment:
This is redundant, the info is displayed below in the item-detail-card just after the searchbar